tests/fixtures/analysis_form_advanced/analysis_form.json

```diff
@@ -20,94 +20,97 @@
       }
     },
     {
-      "type": "select_multiple",
-      "select_from_list_name": "record_a_note_tones",
+      "type": "qual_select_multiple",
       "name": "record_a_note/tone_of_voice",
       "path": [
         "clerk_interactions-record_a_note",
         "tone_of_voice"
       ],
-      "label": [
+      "label": "How was the tone of the clerk's voice?",
+      "labels__for_future_use": [
         "How was the tone of the clerk's voice?",
         "Kiel estis la tono de la voĉo de la oficisto?"
       ],
-      "source": "clerk_interactions-record_a_note"
+      "source": "clerk_interactions-record_a_note",
+      "choices": [
+        {
+          "uuid": "anxious",
+          "labels": {
+            "_default": "Anxious",
+            "en": "Anxious",
+            "es": "Maltrankvila"
+          }
+        },
```

Comment:
Per the KPI schema, choices get embedded directly inside each question, not stored as independent arrays